### ISSUE TITLE
Table of contents not rendering in some cases

### ISSUE THREAD

**Issue by IvanRibakov** [FredGandt/RFCRestyle#3]
Thank you for the great extension! 

It has been working flawlessly for me for more than 3 years but recently I've started experiencing issues with rendering of table of contents. 

Chrome console contains following error message: 

``` 
Error handling response: TypeError: Cannot read properties of null (reading 'textContent') at chrome-extension://babdjpjkdmdppnlgjlpgiknmbdblmdbd/init.js:112:76 
Unchecked runtime.lastError: The message port closed before a response was received. 
``` 

Example URL that doesn't work - https://datatracker.ietf.org/doc/html/rfc3550

**Comment by FredGandt** [comment]
Thank you @IvanRibakov 

I believe I have fixed the problem (version 3.6.1); the GitHub code is updated but the Chrome Web Store extension has to be reviewed by Google before it is pushed to users, so please be patient.

Let me know if the issue is not fixed or if you encounter other issues.

P.S. The error `Unchecked runtime.lastError: The message port closed before a response was received` will persist and is unrelated to the ToC not being created.

**Comment by FredGandt** [comment]
The extension has passed review and is live with the fix. Go to `chrome://extensions/` and check you're running version 3.6.1 and, if not, click the "update" button.
